roles/valheim/defaults/main.yml

```diff
@@ -0,0 +1,54 @@
+---
+valheim_enabled: false
+valheim_available_externally: false
+
+# directories
+valheim_home: "{{ docker_home }}/valheim"
+valheim_data_directory: "{{ valheim_home }}/valheim"
```

Comment:
Should this be `{{ valheim_home }}/data` or similar? Thsi would create a strange directory structure...